app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,27 @@
+class WebhooksController < ApplicationController
+  include ShopifyApp::WebhookVerification
+
+  class ShopifyApp::MissingWebhookJobError < StandardError; end
+
+  def receive
+    job_args = {shop_domain: shop_domain, webhook: webhook}
+    webhook_job_klass.perform_later(job_args)
+    head :ok
+  end
+
+  private
+
+  def webhook
+    params.except(:controller, :action, :type)
+  end
+
+  def webhook_job_klass
+    "#{webhook_type.classify}Job".constantize
```

Comment:
I want this to raise if the constant is not found so I can raise the specific error about the job not existing